exercises/secret-handshake/src/example/kotlin/Signal.kt

```diff
@@ -0,0 +1,5 @@
+enum class Signal {
```

Comment:
I don't believe this duplicate enum is necessary, I believe the `src/main/kotlin` classes are on the classpath.